src/main/webapp/i18n/en/quizExercise.json

```diff
@@ -177,12 +177,14 @@
             },
             "scoringType": {
                 "all_or_nothing": "All or Nothing",
-                "proportional_with_penalty": "Proportional with Penalty"
+                "proportional_with_penalty": "Proportional with Penalty",
+                "proportional_without_penalty": "Proportional without Penalty"
             },
             "scoringTypeExplanation": {
                 "title": "Scoring Types",
                 "all_or_nothing": "All correct choices must be chosen and all wrong choice must be empty to achieve the full score. If a mistake is made the participant achieves 0 points. There is no score in between.",
-                "proportional_with_penalty": "Each correct answer results in a fraction of the total score. To avoid guesswork, the same fraction is subtracted from the achieved points for each mistake. Example: if the score of the question is 3 and there are 5 options. Each correct answer option results in 0.6 points. Each wrong answer option subtracts 0.6 points. A student with 4 correct and 1 wrong choice would then receive 1.8 points."
+                "proportional_with_penalty": "Each correct answer results in a fraction of the total score. To avoid guesswork, the same fraction is subtracted from the achieved points for each mistake. Example: if the score of the question is 3 and there are 5 options. Each correct answer option results in 0.6 points. Each wrong answer option subtracts 0.6 points. A student with 2 correct and 2 wrong answers would then receive 0.6 points.",
```

Comment:
```suggestion
                "proportional_with_penalty": "Each correct answer results in a fraction of the total score. To avoid guesswork, the same fraction is subtracted from the achieved points for each mistake. Example: if the score of the question is 3 and there are 5 options. Each correct answer option results in 0.6 points. Each wrong answer option subtracts 0.6 points. A student with 3 correct and 2 wrong answers would then receive 0.6 points.",
```